edit_course/templates/edit_course/batch_assess.html

```diff
@@ -18,17 +18,17 @@
         <h3 class="panel-title">{% trans "Batch create and assess submissions" %}</h3>
     </div>
     <div class="panel-body">
-        {% blocktrans %}
+        {% blocktrans with profile.id as id %}
 <p><em>Use only if you know what you are doing!</em></p>
 <p>By submitting json to the form below you can try to create and assess multiple exercises for students automatically.</p>
 <dl>
 	<dt>XXXXX1 and XXXXX2</dt>
     <dd>the real student ids</dd>
-	<dt>X1 (integer)</dt>
-    <dd>the system userprofile id of the desired grader</dd>
+	<dt>X1 (integer)</dt> <p>Your userprofile id is <b>{{id}}</b>.</p>
```

Comment:
X1 is now missing the description (dd element). It could be better to add the id inside the dd element like:

`<dd>the system userprofile id of the grader (your id is {{id}}</dd>`